Correct typo and add missing attribute

rreinhardt9 opened the pull request:
## Why

There was a typo in one of the attribute names that has been causing trouble when you copy and paste from this guide for setup.

Additionally, there is a required "user id" attribute that was not documented in this guide that has also caused some pain for initial setups.

## What

Correct the attribute name that should be entered to correspond with user.mail

Add the missing user ID attribute and have it associate with user.objectid

azurecla commented:
Hi __@dstolts__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rmca14 commented:
@elfisher 
Please review this PR and approve/disapprove it here with a comment, thanks.

elfisher commented:
This is a great add. I think we could be more explicit by saying "When creating enabling Geo-Replication in the portal, if you do not already have a server for your secondary, you can create a server on the fly."

rmca14 commented:
@dstolts 
Feel free to make the author's suggested change, thanks.

rmca14 commented:
@dstolts 
If you'd like to make the change as noted, please feel free to do so -- we can't make it for you in this pull request. Or feel free to close this pull request.
@elfisher 
If dstolts closes this pull request, and if you'd like this change made, feel free to open a new pull request (in the private repo) with the change.

Thanks.

rmca14 commented:
Closed due to inactivity.

rmca14 commented:
@elfisher 
Feel free to reopen this PR or open a new PR to make this change.

rreinhardt9 committed:
Correct typo and add missing attribute

Correct the attribute name that should be entered to correspond with user.mail

Add the missing user ID attribute and have it associate with user.objectid
